### PR TITLE
Fix4307:Showing irrelavant warning whenever wrong details are shared

### PR DESCRIPTION
**Pull request by AishwaryJain07**
fixed it don't shows the warning triggered unnecessarily

### CODE REVIEW

**Comment by AishwaryJain07** [comment]
@Suryansh5545 can you please review this!!

**Comment by Suryansh5545** [comment]
@AishwaryJain07 I see two PR open for this issue. Can you please keep only one open and close the other one.

File: frontend/src/views/web/landing.html [hunk 1]
```diff
@@ -32,7 +32,7 @@ <h3 class="text-dark-black"><strong>Evaluating state-of-the-art in AI</strong></
                         <div class="row">
                             <div class="col s12 m12 l12">
                                 <div class="center fs-20 w-500">
-                                    <h4><strong>400+</strong></h4>
+                                    <h4><strong>200+</strong></h4>
```
Comment (Suryansh5545) [inline frontend/src/views/web/landing.html:35]: I think you have added some unrelated commits to this PR @AishwaryJain07

File: frontend/src/views/web/landing.html [hunk 2]
```diff
@@ -48,7 +48,7 @@ <h4><strong>400+</strong></h4>
                         <div class="row">
                             <div class="col s12 m12 l12">
                                 <div class="center fs-20 w-500">
-                                   <h4><strong>40,000+</strong></h4>
+                                   <h4><strong>18,000+</strong></h4>
```
Comment (Suryansh5545) [inline frontend/src/views/web/landing.html:51]: Same here

File: frontend/src/views/web/landing.html [hunk 3]
```diff
@@ -64,7 +64,7 @@ <h4><strong>40,000+</strong></h4>
                         <div class="row">
                             <div class="col s12 m12 l12">
                                 <div class="center fs-20 w-500">
-                                    <h4><strong>350,000+</strong></h4>
+                                    <h4><strong>180,000+</strong></h4>
```
Comment (Suryansh5545) [inline frontend/src/views/web/landing.html:67]: Same

File: frontend/src/views/web/landing.html [hunk 4]
```diff
@@ -80,7 +80,7 @@ <h4><strong>350,000+</strong></h4>
                         <div class="row">
                             <div class="col s12 m12 l12">
                                 <div class="center fs-20 w-500">
-                                    <h4><strong>40+</strong></h4>
+                                    <h4><strong>30+</strong></h4>
```
Comment (Suryansh5545) [inline frontend/src/views/web/landing.html:83]: Same

File: frontend_v2/src/app/components/home/home.component.html [hunk 5]
```diff
@@ -75,7 +75,7 @@ <h3><strong>40,000+</strong></h3>
             <div class="row">
               <div class="col s12 m12 l12">
                 <div class="center fs-20 w-500">
-                  <h3><strong>350,000+</strong></h3>
+                  <h3><strong>180,000+</strong></h3>
```
Comment (Suryansh5545) [inline frontend_v2/src/app/components/home/home.component.html:78]: Here too

File: frontend_v2/src/app/components/home/home.component.html [hunk 6]
```diff
@@ -59,7 +59,7 @@ <h3><strong>400+</strong></h3>
             <div class="row">
               <div class="col s12 m12 l12">
                 <div class="center fs-20 w-500">
-                  <h3><strong>40,000+</strong></h3>
+                  <h3><strong>18,000+</strong></h3>
```
Comment (Suryansh5545) [inline frontend_v2/src/app/components/home/home.component.html:62]: Same here

**Review by Suryansh5545 — COMMENTED** [review]
Left some comments
